host/sgx/calls.c

```diff
@@ -367,22 +367,18 @@ static oe_result_t _handle_call_host_function(
 
 static const char* oe_ocall_str(oe_func_t ocall)
 {
-    static const char* func_names[] = {"CALL_HOST_FUNCTION",
-                                       "GET_QE_TARGET_INFO",
-                                       "GET_QUOTE",
-                                       "GET_REVOCATION_INFO",
-                                       "GET_QE_ID_INFO",
-                                       "THREAD_WAKE",
-                                       "THREAD_WAIT",
-                                       "THREAD_WAKE_WAIT",
-                                       "MALLOC",
-                                       "REALLOC",
-                                       "FREE",
-                                       "WRITE",
-                                       "SLEEP",
-                                       "GET_TIME",
-                                       "BACKTRACE_SYMBOLS",
-                                       "LOG"};
+    // clang-format off
+    static const char* func_names[] =
+    {
+        "CALL_HOST_FUNCTION",
+        "THREAD_WAKE",
```

Comment:
Thread wake and thread wait have been moved to EDL. Haven't they?

Reply:
No, we retained them because they fall under the "trivial" categoery (single 64-bit input value).